site/src/components/site-header.md

```diff
@@ -0,0 +1,499 @@
+---
+title: Site header
+description: The Site header component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - site-header
+---
+
+# Site header
+
+Site header component allows adding full width container to hold everything site's header might need - the logo, brand link, navigation, and other controls.
+
+**This page is best viewed with the content section full screened because banners are intended to be full width elements. Go full screen by clicking the button in the top right corner.**
+
+## Usage
+
+The site header is composed of the following elements:
+
+| Component             | Description                                                 |
+| --------------------- | ----------------------------------------------------------- |
+| `site-header`         | The main container for the site header.                     |
+| `site-header-button`  | Buttons within the site header.                             |
+| `site-header-logo`    | The logo displayed in the site header.                      |
+| `site-header-divider` | A vertical divider between elements.                        |
+| `site-header-brand`   | The brand name or title in the site header.                 |
+| `site-header-nav`     | The navigation container within the site header.            |
+| `site-header-panel`   | A panel that can be expanded to display additional content. |
+
+Below is the example of the site header with logo, brand and navigation links.
+
+```html-no-indent
+<div class="site-header">
+	<!-- Microsoft logo -->
+	<a href="https://www.microsoft.com" aria-label="Microsoft">
```

Comment:
There's really no class here?

Reply:
Nope. `site-header-logo` applied directly to the image. Cause logo is not necessarily a link.

Reply:
I think that's a miss on our end. It's quite odd to me to have a classless direct child of a component. The selector should probably be 
```
site-header-logo > svg

Reply:
Updated!